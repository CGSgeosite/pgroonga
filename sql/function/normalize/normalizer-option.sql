SELECT pgroonga_command('plugin_register normalizers/mysql')::json->1;
SELECT pgroonga_normalize('aBcDe 123', 'NormalizerMySQLGeneralCI');
