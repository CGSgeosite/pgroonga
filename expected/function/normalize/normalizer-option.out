SELECT pgroonga_command('plugin_register normalizers/mysql')::json->1;
 ?column? 
----------
 true
(1 row)

SELECT pgroonga_normalize('aBcDe 123', 'NormalizerMySQLGeneralCI');
 pgroonga_normalize 
--------------------
 ABCDE 123
(1 row)

